src-ts/tools/work/work-status-renderer/WorkStatusRenderer.tsx

```diff
@@ -0,0 +1,22 @@
+import { IconOutline } from '../../../lib'
+import { Work, WorkStatus } from '../work-functions'
+
+import styles from './WorkStatusRenderer.module.scss'
+
+function WorkStatusRenderer(work: Work): JSX.Element {
+
+    const statusKey: (keyof typeof WorkStatus) | undefined = Object.entries(WorkStatus)
```

Comment:
Gonna need some education on this as well.

Reply:
this just gets you the key of the enum rather than the value.

```
enum WorkStatus {
   ready = 'Solutions Ready"
   ...
}
```

in this case, i want the key, which is "ready," rather than the value, which is "Solutions Ready."